views/view.html

```diff
@@ -12,6 +12,10 @@
   <meta name="description" content="{{ seoDescription }}" />
 {% endif %}
 
+{% if data.page.seoRobots and not 'index,follow' %}
```

Comment:
The second part of this conditional can never be false since it's not an available option. Since someone may mistakenly select an indexing option then want to revert to the default, I think we should have that in there as an option (I hope I didn't suggest removing it previously).

Reply:
since index,follow is the default I was trying to not have our meta tag placed and just let the browser default to take over

Reply:
I agree about that. Let's say I'm editing this page and set the field to `noindex`. I then realize that was a mistake and want to go back to the default. Right now I can't do that (you may be able to move the select field back to a null option, but I'm not sure and that's confusing for a lay user).

So this conditional could stay as-is, but by adding a "Default" option to the select choices we let people correct their mistakes.

Reply:
I'm not totally clear what you are looking for but I made some changes which might fulfill your wants.